data/rulei/rulei.txt

```diff
@@ -90,8 +90,30 @@ mission "Rulei Planet"
 			`	It's best you don't remain here for too long.`
 				decline
 
+mission "Rulei: You Are (Not) Alone"
+	invisible
+	landing
+	to complete
+		never
+	to offer
+		has "First Contact: Rulei: offered"
+	on enter "L-118"
+		log `Discovered a region of emptiness east of the core, where systems are shrouded by a vast, eerie darkness. Other than dim stars and a few planets, nothing exists.`
+		conversation
+			`Upon arriving into this system, you notice a strange murkiness curtaining the stars beyond, and your scanners are having trouble making sense of it. The few stellar objects present are eerily blotted out by a vast, nebulous shade. Aside from them, it's as if there is nothing here. Even the shade itself is nothing more than the emptiness of space itself, and throughout it, no one. Nothing.`
 
-mission "Umbral Reach"
+	on enter "L-6181"
+		fail
+		conversation
+			`As you continue through this space of emptiness, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you are met with what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, rather, it appears as some sort of "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`
```

Comment:
I like the flavor of this conversation, but there's some internal inconsistency. 
If the player's scanners don't read the Jje as a lifeform, how do they know to describe it as a "creature"? Does it approach the ship such that FL can see it with their own eyes, and thus confirm its living nature? i.e.
```suggestion
			`As you continue through this space of emptiness, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you catch a glimpse of what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, rather, it appears as some sort of "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`
```
A "'collection' of matter" is equally nebulous: presumably the sensors describe _anything with mass_ as a collection of matter. If it's their inability to describe it that you want to emphasize, maybe something like the following would better convey that?
```suggestion
			`As you continue through this space of emptiness, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you are met with what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, categorizing it only as a mere "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`
```

Reply:
The latter sounds good. As for it being "a creature", I wouldn't put it past the same kind of recognition the player has for something like void sprites, or presumably the Vyrmeid. It's still "appears to be", as it's a moving form of sorts amongst dead systems, so I don't think it's too far fetched.